rllib/utils/tests/test_checkpoint_utils.py

```diff
@@ -153,11 +155,15 @@ def test_msgpack_checkpoint_translation_multi_agent(self):
         def mapping_fn(aid, episode, worker, **kwargs):
             return "pol" + str(aid)
 
-        tune.register_env("ma", lambda _: MultiAgentCartPole(config={"num_agents": 3}))
+        tune.register_env("ma", lambda cfg: RandomMultiAgentEnv(config=cfg))
 
         config = (
             DQNConfig()
-            .environment("ma")
+            .environment("ma", env_config={
```

Comment:
Nit: We only test the env config here. Not `AlgorithmConfig.action_space` and `AlgorithmConfig.observation_space`.
Can we add this?

Reply:
Great point, will fix.